package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sanctuary-def",
+  "version": "0.0.0",
+  "description": "Define strict, curried functions",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plaid/sanctuary-def.git"
+  },
+  "dependencies": {
+    "ramda": "0.18.x"
```

Comment:
No rush from this end.  Got plenty keeping me too busy to play more with JS + Ramda as it is.  Will have to wait to add Sanctuary-def to that mix anyway.  Enjoy the holiday :smile: